tests/manager/bug1163-001.phpt

```diff
@@ -0,0 +1,85 @@
+--TEST--
+PHPC-1163: Unacknowledged write concern should omit implicit session
+--XFAIL--
+Depends on CDRIVER-2615
```

Comment:
Tests pass locally after applying the patch from [CDRIVER-2615](https://jira.mongodb.org/browse/CDRIVER-2615), but feel free to verify on your own. In any event, that fix won't show up until libmongoc 1.10.